opentech/public/people/models.py

```diff
@@ -13,13 +13,12 @@
     FieldRowPanel,
     InlinePanel,
     MultiFieldPanel,
-    PageChooserPanel,
     StreamFieldPanel
 )
 from wagtail.wagtailimages.edit_handlers import ImageChooserPanel
 
 from opentech.public.utils.blocks import StoryBlock
-from opentech.public.utils.models import BasePage
+from opentech.public.utils.models import BasePage, BaseFunding, FundingMixin
```

Comment:
since this is shared, should utils be... on the same level of apply and public?

Reply:
It is only shared in the public section of the site so didn't want it higher. equally it seems a bit weird in the utils folder.